fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -449,6 +470,7 @@ class BrowserRobot {
     }
 
     fun verifySuggestedUserName(userName: String) {
+        Log.i(TAG, "verifySuggestedUserName: Looking for suggested logins prompt")
         mDevice.findObject(
             UiSelector()
                 .resourceId("$packageName:id/mozac_feature_login_multiselect_expand"),
```

Comment:
If you'd replaced this with itemWithResId, I think that already contains this kind of logging.

Reply:
Done !